arc-0031/README.md

```diff
@@ -0,0 +1,95 @@
+---
+arc: 0031
+title: Improving mappings
+authors: evan@demoxlabs.xyz, mike@demoxlabs.xyz
+discussion: [ARC-0031: Improving mappings](https://github.com/AleoHQ/ARCs/discussions/20)
+topic: Language
```

Comment:
Formatting to follow https://github.com/AleoHQ/ARCs/tree/master/arc-0001#specification
```suggestion
topic: Application
```